From 5604e98ec3fe4dc6d5558e213999324d246f9463 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Fri, 2 Dec 2022 15:41:29 +0530
Subject: [PATCH] Add mediatranscode sepolicy rules to public.

As mediatranscode update in google moving from private to public

OAM-105142
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/groups/media/auto/BoardConfig.mk b/groups/media/auto/BoardConfig.mk
index f348cef..ef36fe8 100644
--- a/groups/media/auto/BoardConfig.mk
+++ b/groups/media/auto/BoardConfig.mk
@@ -1,4 +1,5 @@
 SYSTEM_EXT_PRIVATE_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/system_ext/private
+SYSTEM_EXT_PUBLIC_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/system_ext/public
 
 {{#enable_msdk_omx}}
 INTEL_STAGEFRIGHT := true
-- 
2.38.1

